sympy/geometry/polygon.py

```diff
@@ -438,6 +438,70 @@ def second_moment_of_area(self, point=None):
         return I_xx, I_yy, I_xy
 
 
+    def polar_modulus(self):
+        """Returns the polar modulus of a two-dimensional polygon
+
+        Examples
+        ========
+
+        >>> from sympy import Polygon, symbols
+        >>> a, b = symbols('a, b')
+        >>> rectangle = Polygon((0, 0), (a, 0), (a, b), (0, b))
+        >>> rectangle.polar_modulus()
+        a**3*b/12 + a*b**3/12
+        """
+        return self.second_moment_of_area()[0] + self.second_moment_of_area()[1]
+
+
+    def section_modulus(self, point=None):
+        """Returns a tuple with the section modulus of a two-dimensional
+        polygon.
+
+        Parameters
+        ==========
+
+        point : Point, two-tuple of sympifyable objects, or None(default=None)
+            point is the point at which section modulus is to be found.
+            If "point=None" it will be calculated for the point farthest from the
+            centroidal axis of the polygon.
+
+        Returns
+        =======
+
+        S_x, S_y: numbers or SymPy expressions
+                  S_x is the section modulus wrt the x-axis
+                  S_y is the section modulus wrt the y-axis
+                  A negetive sign indicates that the section modulus is
+                  determined for a point below the centroidal axis
+
+        Examples
+        ========
+
+        >>> from sympy import symbols, Polygon, Point
+        >>> a, b = symbols('a, b', positive=True)
+        >>> rectangle = Polygon((0, 0), (a, 0), (a, b), (0, b))
+        >>> rectangle.section_modulus()
+        (a*b**2/6, a**2*b/6)
+        >>> rectangle.section_modulus(Point(a/4, b/4))
```

Comment:
Shouldn't we return  a `ValueError` if  the `Point` lies outside of the `Polygon`?
```
>>> rectangle = Polygon((0, 0), (a, 0), (a, b), (0, b))
>>> rectangle.section_modulus(Point(10*a, 10*b))
(a*b**2/114, a**2*b/114)
```

Reply:
But I guess then it won't be able to handle arbitrary points like:
 `rectangle.section_modulus(Point(x, y))`
 I have added such an example in the tests.